hack/verify-golint.sh

```diff
@@ -94,18 +100,29 @@ fi
 
 # check to make sure all packages that pass lint are in the linted file.
 echo
-if [ ${#linted[@]} -eq 0 ]; then
+if [ ${#linted[@]} -eq 0 -a ${#gone[@]} -eq 0 ]; then
```

Comment:
@stevekuznetsov does this do what it's supposed to do?  It doesn't have to be the pinnacle of beauty, it just needs to work.